### DISCUSSION TITLE
Can I use github pages for commercial purpose?

### DISCUSSION THREAD

**Discussion by KunwarAhlawat** [community/community#37435] [Other Feature Feedback, Questions, & Ideas]
Hi! I am Kunwar Ahlawat. I own a Web Design and Development Agency. I have a question related to GitHub Pages. Can I host a commercial website or web pages on GitHub pages?

**Comment by radar-blip** [comment]
You may find some useful information here: https://webapps.stackexchange.com/questions/56898/am-i-allowed-to-host-a-commerical-website-on-github-pages

**Reply by KunwarAhlawat** [reply]
Thanks

**Comment by airtower-luna** [comment]
The Pages documentation says this under ["Prohibited uses"](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#prohibited-uses):

> GitHub Pages is not intended for or allowed to be used as a free web-hosting service to run your online business, e-commerce site, or any other website that is primarily directed at either facilitating commercial transactions or providing commercial software as a service (SaaS).
